### PR TITLE
fix(suite-native): fix biometrics

### PR DESCRIPTION
**Pull request by vytick**
biometrics enabling worked correctly from settings but not from initial dialog. Due to delayed set of `isBiometricsOptionEnabled`. This solves the issue.

### CODE REVIEW

File: suite-native/biometrics/src/useBiometricsSettings.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export const useBiometricsSettings = () => {
 
     const toggleBiometricsOption = useCallback(async (): Promise<BiometricsToggleResult> => {
         const isBiometricsAvailable = await getIsBiometricsFeatureAvailable();
+        const isEnabled = isBiometricsOptionEnabled;
```
Comment (Nodonisko) [inline suite-native/biometrics/src/useBiometricsSettings.ts]: use `isBiometricsOptionEnabled` directly

File: suite-native/module-home/src/components/BiometricsBottomSheet.tsx [hunk 2]
```diff
@@ -57,9 +57,12 @@ export const BiometricsBottomSheet = () => {
     const [isVisible, setIsVisible] = useState(false);
     const [biometricsTypes, setBiometricsTypes] = useState<AuthenticationType[]>([]);
 
-    const isFace = biometricsTypes.filter(biometricsType =>
-        [AuthenticationType.FACIAL_RECOGNITION, AuthenticationType.IRIS].includes(biometricsType),
-    );
+    const isFace =
```
Comment (Nodonisko) [inline suite-native/module-home/src/components/BiometricsBottomSheet.tsx]: Can you make this simpler? Seems uneccasary complicated, use pipe operator and some functions from here https://mobily.github.io/ts-belt/api/array

**Review by Nodonisko — CHANGES_REQUESTED** [review]

File: suite-native/module-home/src/components/BiometricsBottomSheet.tsx (hunk 2 above)
Comment (Nodonisko) [reply suite-native/module-home/src/components/BiometricsBottomSheet.tsx]: `biometricsTypes.some(biometricsType => [AuthenticationType.FACIAL_RECOGNITION, AuthenticationType.IRIS].includes(biometricsType));`

Comment (vytick) [reply suite-native/module-home/src/components/BiometricsBottomSheet.tsx]: f9d6552

File: suite-native/biometrics/src/useBiometricsSettings.ts (hunk 1 above)
Comment (vytick) [reply suite-native/biometrics/src/useBiometricsSettings.ts]: f9d6552

**Review by Nodonisko — CHANGES_REQUESTED** [review]
@PeKne Can you review this since you are more familiar with biometrics than I am?

**Review by Nodonisko — APPROVED** [review]

**Review by PeKne — APPROVED** [review]

**Pull request merged**